fotoapparat/src/main/java/io/fotoapparat/parameter/SupportedParameters.kt

```diff
@@ -56,10 +56,11 @@ internal class SupportedParameters(
     }
 
     /**
-     * @see Camera.Parameters.isZoomSupported
+     * @return [io.fotoapparat.parameter.Zoom.FixedZoom] if [Camera.Parameters.isZoomSupported] returns false,
+     * and [io.fotoapparat.parameter.Zoom.VariableZoom] with max zoom level otherwise.
      */
     val supportedZoom by lazy {
-        cameraParameters.isZoomSupported
+        if (cameraParameters.isZoomSupported) Zoom.VariableZoom(0, cameraParameters.maxZoom) else Zoom.FixedZoom
```

Comment:
`0` is magic number so let's extract it into a const somewhere in this file

Reply:
btw if the minValue is alway 0 we can only return maxZoom in the `VariableZoom` and add in the doc that levels are between `0 and max`

Reply:
According to the Camera.Parameters docs, "setZoom" method can use values in the range [0, maxZoom], so I think min value always 0. 

Let's remove min value =)

Reply:
let's just add a small note in the Variable zoom that the min value is always 0